pkg/rbd/controllerserver.go

```diff
@@ -129,6 +129,36 @@ func (cs *ControllerServer) parseVolCreateRequest(ctx context.Context, req *csi.
 	return rbdVol, nil
 }
 
+func (cs *ControllerServer) getExistingVolume(ctx context.Context, req *csi.CreateVolumeRequest, rbdVol *rbdVolume) (*csi.CreateVolumeResponse, error) {
```

Comment:
can we update the function name to some more meaningful, as it's not getting the existingVolume as function name says, this can be a function, not a method as it won't use `cs`

Reply:
Ok, do you like `buildCreateVolumeResponse()` as name of the function? That is what will be included in the next update.